src/static/riot/competitions/detail/leaderboards.tag

```diff
@@ -38,7 +43,13 @@
         self.hidden_column_keys = []
 
         self.get_score = function(column, submission) {
```

Comment:
should we process the scores once and store them in a dictionary or something? this is performant?

Reply:
There is definitely a better way to do this, but I'm thinking we'll need some function here if we want to be able to re-order columns later. What are you thinking?

Reply:
I actually had this possibility in mind when making my pr. If we would rather process this all on the backend then serve one JSON for the leaderboards we could modify this code a little. https://github.com/codalab/competitions-v2/pull/499/files#diff-ee219ec77307c8c2bd9bb7ab68227f64R286-R289
Or we could create a new method for the front-end that is separate from the api call.